qiita_db/metadata_template/constants.py

```diff
@@ -70,6 +70,15 @@
 ALL_RESTRICTIONS = [SAMPLE_TEMPLATE_COLUMNS, PREP_TEMPLATE_COLUMNS,
                     PREP_TEMPLATE_COLUMNS_TARGET_GENE]
 
+# This is what we consider as "NaN" cell values on metadata import
+NA_VALS = ['', 'no_data', 'unknown', 'Unspecified']
```

Comment:
Uhm, I don't think that is correct. I think we want to maintain `XXQIITAXX` as those are indicative that they need to change those columns.